tests/core/test_config_substitutor.py

```diff
@@ -0,0 +1,261 @@
+from contextlib import contextmanager
+from unittest import mock
+
+import pytest
+
+from great_expectations.core.config_substitutor import (
+    ConfigurationSubstitutor,
+    secretmanager,
+)
+
+
+@pytest.fixture
+def config_substitutor() -> ConfigurationSubstitutor:
+    return ConfigurationSubstitutor()
+
+
+@contextmanager
+def does_not_raise():
+    yield
+
+
+@pytest.mark.parametrize(
+    "input_value,method_to_patch,return_value",
```

Comment:
These are all tests that previously lived in `tests/data_context/test_data_context_utils.py`